tests/c_languages/cppcheck_test_files/bad_file.cpp

```diff
@@ -1,8 +0,0 @@
-#define f(c) { \
```

Comment:
why removing this file?

Reply:
I have replaced this `bad_file.cpp` with `bad_file.c` and made some changes to ensure that `cppcheck` working on both C and  C++ files :)
Do you want me to keep the same content in `bad_file.c`?

Reply:
if possible yes, please keep old test files and create new ones if you want to test new aspects :+1:

Reply:
Done! Is it fine now?I have removed `bad_file.c` because it had the same aspect as `bad_file.cpp`